src/TropicalGeometry/matrix.jl

```diff
@@ -74,3 +74,30 @@ function is_tropically_generic(A::MatrixElem{<:TropicalSemiringElem})
     return helper(transpose(A),nca,nra)
   end
 end
+
+@doc raw"""
+    is_tropically_generic(A::MatrixElem{<:TropicalSemiringElem})
```

Comment:
This docstring does not fit the function it is attached to

Reply:
Sorry, this is my first time creating a pull request and I didn't submit the most recent changes. It should be 'correct' now.